vllm/engine/arg_utils.py

```diff
@@ -439,6 +440,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             type=int,
                             default=EngineArgs.tensor_parallel_size,
                             help='Number of tensor parallel replicas.')
+        parser.add_argument('--data-parallel-size',
+                            '-dp',
+                            type=int,
+                            default=EngineArgs.data_parallel_size,
+                            help='Number of data parallel replicas.')
```

Comment:
Could you add to the help string that MoE layers will be sharded according to the product of DP and TP?

Reference discussion here https://github.com/vllm-project/vllm/pull/14305#discussion_r1983588471